### DISCUSSION TITLE
Imgui for arm64

### DISCUSSION THREAD

**Discussion by tacticalyash** [ocornut/imgui#7719] [New Users Build/Link/Run issues ONLY!]
### Version/Branch of Dear ImGui:

Version 1.XX, Branch: XXX (master/docking/etc.)

### Back-ends:

imgui_impl_XXX.cpp + imgui_impl_XXX.cpp

### Compiler, OS:

Ubuntu arm 64

### Full config/build information:

We tried for the arm 64 version with cmake , its not getting build any solutions.

### Details:

**My Issue/Question:**

XXX _(please provide as much context as possible)_

### Minimal, Complete and Verifiable Example code:

```cpp
// Here's some code anyone can copy and paste to reproduce your issue
ImGui::Begin("Example Bug");
MoreCodeToExplainMyIssue();
ImGui::End();
```

**Comment by ocornut** [comment]
Dear ImGui should build on every modern CPU. 
you are not providing any details about your issue.

**Reply by tacticalyash** [reply]
It's  fixed now , I was not adding the .0 files Thank you !

**Comment by ocornut** [comment]
Moved from issue #7718 which ignored every template and guidelines. Please don’t do this, thank you.
